### ISSUE TITLE
Position

### ISSUE THREAD

**Issue by LinkOut** [jakob-grabner/Circle-Progress-View#45]
Is it possibile to set some margin to unit and amount of %, since is not centered.

Thanks

**Comment by bzzo** [comment]
use custom:cpv_textMode="Text" and cpv_text="TEXT%" instead of cpv_value="12"

**Comment by LinkOut** [comment]
Thanks for the answer, what do you mean i dont understand

**Comment by bzzo** [comment]
i had a problem to center text when i use value and unit like 15% (setValue(15) and showUnit(true) and setUnit(%))
so: i changed textmode to TEXT and setText(15%) showUnit(false).

now i see text centered

**Comment by LinkOut** [comment]
Yes i got this problem coz the unit is not centered in the circle. Thanks for your solution, but i need to use unit
